django_mongodb_backend/gis/features.py

```diff
@@ -49,10 +57,13 @@ def django_test_skips(self):
                     # Error: Index already exists with a different name
                     "gis_tests.geoapp.test_indexes.SchemaIndexesTests.test_index_name",
                 },
-                "GIS lookups not supported.": {
-                    "gis_tests.geoapp.tests.GeoModelTest.test_gis_query_as_string",
+                "GIS Union not supported.": {
                     "gis_tests.geoapp.tests.GeoLookupTest.test_gis_lookups_with_complex_expressions",
                 },
```

Comment:
Note to self that we should add `@skipUnlessDBFeature("supports_Union_function")` and send the patch upstream. I'll hold off in case any more similar changes are identified by this PR.